examples/api/lib/cupertino/sheet/cupertino_sheet.3.dart

```diff
@@ -0,0 +1,174 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+
+/// Flutter code sample for [CupertinoSheetRoute] with restorable state.
+/// Do we include?
```

Comment:
I don't think showCupertinoSheet can be used with restorable state, because it uses a regular push internally. [showDatePicker](https://api.flutter.dev/flutter/material/showDatePicker.html) works the same way.